src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -16,5 +16,8 @@ export default function (mediaFeature) {
   // SCSS or Less interpolation
   if (hasInterpolation(mediaFeature)) { return false }
 
+  // SCSS variables
+  if (/\$.+?/.test(mediaFeature)) { return false }
```

Comment:
@davidtheclark you are right, we should change also on `/[^\\]\$\(.+?\)/`
And why we support `postcss-simple-vars`? It is not `scss`, `less` and `sugarcss` and not `cssnext`(including postcss-plugins for `css` `draft`), it is custom postcss plugin :smile: with rare usage, may be we should drop this?